pkg/orchestration/wiring/entangler.go

```diff
@@ -99,13 +100,12 @@ func (en *Entangler) Entangle(state *orch_v1.State, context *orchestration.Entan
 		return nil, false, err
 	}
 
-	tags := make(map[voyager.Tag]string, len(serviceProperties.UserTags))
+	tags := make(map[voyager.Tag]string, len(serviceProperties.UserTags)+6)
```

Comment:
why 6? is this optimization worth it? just create an empty map instead?
alternatively, create a separate "static" map with all 6 known elements, i.e.
```go
platformTags := map[voyager.Tag]string{
  en.TagNames.ServiceNameTag: string(context.ServiceName),
  en.TagNames.BusinessUnitTag: serviceProperties.BusinessUnit,
  ...
}
tags := make(map[voyager.Tag]string, len(serviceProperties.UserTags)+len(platformTags))
...
```

Reply:
+6 because there are 6 more elements that are added to the map below.

Reply:
so we will have to update this magic number every time we add/remove tag. see my suggestion to create a separate `platformTags` map and allocate `tags` size based on it (and copy platform tags from it).

Reply:
I don't want to allocate 2 temporary maps just to merge them into a third one. I'll just remove the number if you don't like it so much =)

Reply:
up to you :)